UtilizeSkylineDataServices/Readme.md

```diff
@@ -23,104 +23,145 @@ From the panel of the web app, you will be able to:
+2.	On the **Document** tab, select **Web Server** as the Build target.
+3.	For the **Output** type, select **Package**.
+4.	In the **Distribution** document, click the **Add Application/Library** button.
+5.	In the **Add Application/Library** dialog box, place a checkmark next to the application or library you want to distribute and click **OK**.
+6.	On the **Document** tab, enter values in **Company** and **Product**.
+7.	In **Output** path, set the location to save the distribution output.
+8.	In the **NI Package** section, enter information about the distribution output.
+9.	Click **File»Save** all to save all files.
+10.	Click **Build distribution**. The Build Queue tab shows the status of the build process.
+11.	Open a web browser and go to [Web apps](https://www.systemlinkcloud.com/web-apps).
+12.	To upload your web application, click **Choose .NIPKG** and select the package you just built.
+13.	Once the upload is complete, click on your web app to interact with it.
 
 # On-Premises Web Server
-In this section, you’ll learn how to setup, build, and connect your web application to an on-premises web server.
+In this section, you’ll learn how to set up, build, and connect your web application to an on-premises web server.
 
 ## What You Need To Connect To an On-Premises Web Server from LabVIEW NXG
 ![Configure GVI in LabVIEW vs Hosting on premises](https://ni.github.io/webvi-examples/UtilizeSkylineDataServices/on-premises-config.PNG)
 
-_Figure: Note the input differentiation between using on-premises data services from within LabVIEW vs hosting within the NI Web Server._
+_Figure: Note the input differentiation between connecting to an  on-premises server from LabVIEW or LabVIEW NXG 2.1 Web Module vs. hosting your web application on the NI Web Server._
 
-When communicating with on-premises data services from LabVIEW NXG, you must include a **server url** (i.e. https://systemlinkcloud.com), a **username**, and a **password**. The username and password can be managed with the NI Web Server Configuration utility. This utility can be used to create new users and groups as well as leverage existing LDAP or Windows user accounts.  
+To connect to an on-premises web server from LabVIEW NXG 2.1 Web Module, you must include a **server url** (i.e. https://systemlinkcloud.com), a **username**, and a **password**. The username and password can be managed with the NI Web Server Configuration utility. This utility can be used to create new users and groups as well as leverage existing LDAP or Windows user accounts.  
 
-When the web application is hosted on the NI Web Server, the server url, username, and password are not needed and should not be used. This deliberately minimizes security vulnerabilities.
+When a web application is hosted on an on-premises web server, leave the **server url**, **username**, and **password** inputs empty to minimize security vulnerabilities.
```

Comment:
This is only true if the Web application is hosted by the SystemLink web server. If they are using their own Web server they will need to include credentials to authenticate with the SystemLink Cloud/Server that is hosting the data services they are accessing.